ee/dtr/user/manage-images/sign-images/trust-with-remote-ucp.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Using Docker Content Trust with a Remote UCP Cluster
+description: Learn how to use a single DTR's trust data with remote UCPs.
+keywords: registry, sign, trust, notary
+---
+
+For more advanced deployments, you may want to share one Docker Trusted Registry
+across multiple Universal Control Planes. However, customers wanting to adopt
+this model, alongside the [Only Run Signed
+Images](../.../../ucp/admin/configure/run-only-the-images-you-trust.md) feature
+of UCP, run into problems as each Universal Control Plane operates an
+independent set of users.
+
+Docker Content Trust with a Remote UCP gets around this problem, as User's from
+a remote UCP are able to sign images in the central DTR, and still apply runtime
+enforcement.
+
+In the following example we will connect DTR to 1 remote UCP cluster, however
+this process can be repeated over and over with more remote UCPs. 
+
+![](../../../images/remoteucp-graphic.png)
+
+> Before attempting this guide, please familiarize yourself with [Docker Content
+> Trust](engine/security/trust/content_trust/#signing-images-with-docker-content-trust)
+> and [Only Run Signed
+> Images](../.../../ucp/admin/configure/run-only-the-images-you-trust.md) on a
+> single UCP. A lot of the concepts within this guide may be new without that
+> background. 
+
+## Prerequisites
+
+- Cluster 1, running UCP 3.0.x or higher, with a DTR 2.5.x or higher deployed
+  within the cluster.
+- Cluster 2, running UCP 3.0.x or higher, it is expected that
+  there is no DTR installed on this environment.
+- Nodes on Cluster 2 need to trust the Certificate Authority that signed DTR's
+  TLS Certificate. This can be tested by logging on to a Cluster 2 virtual
+  machine and running `curl https://dtr.example.com`.
+- The DTR TLS Certificate needs be properly configured, ensuring that the
+  **Loadbalancer/Public Address** field has been configured, with this address
+  included [within the
+  certificate](../../../admin/configure/use-your-own-tls-certificates/).
+- A workstation with the [Docker Client](/ee/ucp/user-access/cli/) (CE 17.12 /
+  EE 1803 or newer) installed, as this contains the relevant `$ docker trust`
+  commands. 
+
+## Registering DTR with a remote Universal Control Plane
+
+As there is no registry running within Cluster 2, by default UCP will not know
+where to check for trust data. Therefore, the first thing we need to do is
+register DTR within the Universal Control Plane of Cluster 2. When you normally
+install Docker Trusted Registry, this registration process happens by default to
+a local UCP.
+
+> The registration process allows the remote UCP to get signature data from DTR,
+> however this will not provide Single Sign On, Users on Cluster 2 will not be
+> synced with Cluster 1's Universal Control Plane or Docker Trusted Registry.
+> Therefore when pulling images, if the repository is private, registry
```

Comment:
Do you think we should put a link here back to the registry authentication for both Swarm and Kubernetes deployments?